packages/xo-web/src/xo-app/backup-ng/index.js

```diff
@@ -67,6 +67,9 @@ const SchedulePreviewBody = ({ item: job, userData: { schedulesByJob } }) => (
         </td>
         <td>
           <ActionButton
+            btnStyle='primary'
+            data-id={job.id}
+            data-schedule={schedule.id}
             handler={_runBackupNgJob}
             handlerParam={{
```

Comment:
Remove `handlerParam` if you use `data-*` props.
https://github.com/vatesfr/xen-orchestra/blob/9fb0f793b2150b270072d39316cd8aa6a07a4c3e/packages/xo-web/src/common/action-button.js#L31-L32